app/controllers/reports/regions_controller.rb

```diff
@@ -78,9 +78,18 @@ def details
     else
       [@region, @region.facility_regions].flatten
     end
+
     @repository = Reports::Repository.new(regions, periods: @period_range, follow_ups_v2: current_admin.feature_enabled?(:follow_ups_v2))
     chart_repo = Reports::Repository.new(@region, periods: chart_range, follow_ups_v2: current_admin.feature_enabled?(:follow_ups_v2))
 
+    district_regions = if @region.state_region?
+      [@region, @region.district_regions].flatten
+    end
+
+    if @region.state_region?
+      @district_repository = Reports::Repository.new(district_regions, periods: @period_range, follow_ups_v2: current_admin.feature_enabled?(:follow_ups_v2))
+    end
+
```

Comment:
This is to show call results grouped by district in the state report, this was requested explicitly.